<launch>
  <node pkg="perception_monitor" output="screen" type="covariance.py" name="sensor_covariance_node" respawn="False">
    <param name="measure_lidar" value="true"/>
    <param name="measure_camera" value="false"/>
    <param name="match_distance" value="1.0"/>
    <param name="batch_size" value="20"/>
    <param name="convergence_threshold" value="0.00000001"/>

    <param name="ground_truth_x" value="0.0"/>
    <param name="ground_truth_y" value="0.0"/>
    <param name="ground_truth_z" value="0.0"/>

    <remap from="/input/lidar_observations" to="/ugr/car/observations/lidar"/>
    <remap from="/input/camera_observations" to="/ugr/car/observations/camera"/>
    <remap from="/output/topic" to="/diagnostics/covariances"/>
  </node>
</launch>